cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -379,6 +378,7 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
       showDashboard: true,
       showReports: true,
       showDataPrep: true,
+      // showWranglerDatamodelViewer: true,
```

Comment:
Assuming this is for testing in local development env, it would be great to avoid landing these code into upstream code base. I would remove it in this PR and only keep it in my local branch.

Reply:
Ok, I will remove it.

Reply:
Done.